features/step_definitions/user_steps.rb

```diff
@@ -0,0 +1,17 @@
+Given(/^I fill in "([^"]*)" with "([^"]*)"$/) do |item, content|
+  fill_in item, with: content
+end
+
+Given("the following user exists") do |table|
+  table.hashes.each do
+    FactoryGirl.create(:user, password: 'password')
+  end
+  end
+
+Given("I should see link {string}") do |string|
+  expect(page).to have_link string
+end
+
+Given("I should not see link {string}") do |string|
+  expect(page).not_to have_link string
+end
```

Comment:
It feels like we've already have a lot of similar step definitions like this in other files. Try to reuse as much as you can

Reply:
I don't see another step defining NOT to see a link in the steps

Reply:
Okay

Reply:
Please format your codes